docker-entrypoint.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+cd /usr/src/todo_app/todo_app
+
+$HOME/.poetry/bin/poetry run gunicorn --bind 0.0.0.0:5000 --workers=2 'app:create_app()'
```

Comment:
Is there any particular reason you step one level further into your app at this stage compared to e.g. dev? I can't see any harm but for consistency (and potentially even just relying on your WORKDIR) we could update the Gunicorn command to work from one level higher:
```suggestion
cd /usr/src/todo_app

$HOME/.poetry/bin/poetry run gunicorn --bind 0.0.0.0:5000 --workers=2 'todo_app.app:create_app()'
```